cypress/support/index.js

```diff
@@ -1,16 +1,33 @@
+import '@cypress/react/support'
+import 'givens/setup'
 import './commands'
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-require('cypress-terminal-report/src/installLogsCollector')()
+import { unmount } from '@cypress/react'
+
+try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    require('cypress-terminal-report/src/installLogsCollector')()
+} catch {}
 
 beforeEach(() => {
-    cy.visit('/')
+    if (Cypress.spec.specType === 'component') {
+        // Freeze time to 2021.01.05
+        cy.clock(1609804800000, ['Date'])
```

Comment:
This was causing timezone issue because i think the component funcs/logic will use moment and default to the local timezone. When I moved this time so that EDT is within the same day, it works

Reply:
Improved the timestamp.